packages/block-library/src/list/save.js

```diff
@@ -4,10 +4,10 @@
 import { RichText } from '@wordpress/block-editor';
 
 export default function save( { attributes } ) {
-	const { ordered, values } = attributes;
+	const { ordered, values, reversed, start, type } = attributes;
 	const tagName = ordered ? 'ol' : 'ul';
 
 	return (
-		<RichText.Content tagName={ tagName } value={ values } multiline="li" />
+		<RichText.Content tagName={ tagName } value={ values } reversed={ reversed } start={ start } type={ type } className={ ordered ? 'ol-type-is-' + type : undefined } multiline="li" />
```

Comment:
See https://github.com/WordPress/gutenberg/pull/15113/files#r278709806.
I thought the class was not needed on the front-end.

Reply:
Class is needed to suppress the admin CSS rule in the preview page. https://github.com/WordPress/gutenberg/pull/15113#discussion_r279072304